src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -707,7 +748,7 @@ public void onChange(final List<ResourceChange> changes) {
             log.debug("onChange, type={}, path={}", rc.getType(), path);
 
             // don't care for system area
-            if (path.startsWith(JCR_SYSTEM_PREFIX)) {
+            if (path.startsWith(JCR_SYSTEM_PATH + '/')) {
```

Comment:
while this is not a big deal, why did you change this from a constant to an expression that is executed each time?

Reply:
That is an oversight: I think I inlined a few values, I'll change it back to a constant.